### ISSUE TITLE
/bin/sh: 1: Syntax error: Missing when web-ssh into any pods

### ISSUE THREAD

**Issue by fangli** [astefanutti/kubebox#25]
If I run kubebox in kubernetes as a service, when web-ssh into a pod, it gives an error:

```
/bin/sh: 1: Syntax error: Missing '))'
command terminated with non-zero exit code: Error executing in Docker Container: 2
Disconnected
Type Ctrl-C to close
```

I'm not pretty sure if it's caused by this commit:
https://github.com/astefanutti/kubebox/commit/b795966f51cbf503f2c371ea6fa1579136fbe32b

**Comment by astefanutti** [comment]
Can you confirm you've deployed Kubebox with the `kubernetes.yaml` template? If yes, could you try changing the container image to `astefanutti/kubebox:0.3.1-server` so that we can confirm it's caused by b795966f51cbf503f2c371ea6fa1579136fbe32b?

**Comment by astefanutti** [comment]
Thanks a lot. I suspect char escaping issue too, with '&' messing up the request. What browser do you use? Would you be able to inspect the request being sent in the dev console?

**Comment by astefanutti** [comment]
I've been able to reproduce, both in Web and terminal versions. It should be fixed with 58a35b35617b382480ac264a75fa30e090a70ccb. Thanks for your help!

**Comment by fangli** [comment]
Oh you r the genius! it took me several hours to figure out why that happens. I tried many shell with your getent commands but nothing found.
That's the kubebox version of "rm -rf / tmp/files" lol :)
